syncer/syncer.go

```diff
@@ -2100,8 +2100,6 @@ func (s *Syncer) trackDDL(usedSchema string, sql string, tableNames [][]*filter.
 		shouldSchemaExist = true
 	case *ast.DropTableStmt:
 		shouldExecDDLOnSchemaTracker = true
-		shouldSchemaExist = true
```

Comment:
Do we need to handle `shouldSchemaExist` for `case *ast.DropDatabaseStmt`?

Reply:
We create it if not exist below. But actually no need. Address in 56bf87f